### PR TITLE
Refactor milvus dataprep and retriever

### PR DESCRIPTION
**Pull request by bjzhjing**
Refactor the Milvus Dataprep and Retriever modules to enhance code readability and facilitate user-friendly configuration.

### CODE REVIEW

File: comps/dataprep/milvus/langchain/config.py [hunk 1]
```diff
@@ -3,16 +3,16 @@
 
 import os
 
-# Embedding model
-TEI_EMBEDDING_MODEL = os.getenv("EMBED_MODEL", "maidalun1020/bce-embedding-base_v1")
-# Embedding endpoints
+# Local Embedding model
+LOCAL_EMBEDDING_MODEL = os.getenv("LOCAL_EMBEDDING_MODEL", "maidalun1020/bce-embedding-base_v1")
```
Comment (XuhuiRen) [inline comps/dataprep/milvus/langchain/config.py:7]: what if there is no "LOCAL_EMBEDDING_MODEL"?

**Review by XuhuiRen — APPROVED** [review]

File: comps/dataprep/milvus/langchain/config.py (hunk 1 above)
Comment (bjzhjing) [reply comps/dataprep/milvus/langchain/config.py:7]: It will use the default value "maidalun1020/bce-embedding-base_v1".

**Comment by bjzhjing** [comment]
@XinyuYe-Intel Hi, could you please help review this PR?

**Review by XinyuYe-Intel — APPROVED** [review]

**Comment by bjzhjing** [comment]
@chensuyue Hi Suyue, I checked the CI failures.

1. For Microservice-test(dataprep_milvus_langchain, ...), the environment parameter is changed from MILVUS to MILVUS_HOST for dataprep_milvus_langchain, but this parameter seems to be hardcoded in CI, could you please update it?
2. For Microservice-test(daaprep_pinecone_langchain, ...) it should not be affected this PR.

**Comment by bjzhjing** [comment]
@XinyuYe-Intel please help merge this PR, thanks!

**Comment by XinyuYe-Intel** [comment]
> @XinyuYe-Intel please help merge this PR, thanks!

please make sure to pass all tests.

**Comment by bjzhjing** [comment]
> > @XinyuYe-Intel please help merge this PR, thanks!
> 
> please make sure to pass all tests.

That is due to the hard code in CI script as I mentioned above. Could anyone from CI side take a look? @daisy-ycguo @chensuyue

**Comment by lvliang-intel** [comment]
@bjzhjing,
Please check the CI issue,  microservice test reported errors.

**Comment by bjzhjing** [comment]
> @bjzhjing, Please check the CI issue, microservice test reported errors.

@lvliang-intel @XinyuYe-Intel Hi, the test script is modified and CI has passed.

**Pull request merged**